lib/List.js

```diff
@@ -26,6 +55,22 @@ class List extends React.Component {
     return itemTypes;
   }
 
+  checkForModificationsAndReturnData(data) {
+    const { groupBy, sortBy } = this.props;
+
+    if (groupBy && sortBy) {
```

Comment:
if statements that modify a let variable in this order sort, group, and then return.